src/main/java/emu/grasscutter/server/event/game/SceneMetaLoadEvent.java

```diff
@@ -0,0 +1,15 @@
+package emu.grasscutter.server.event.game;
+
+import emu.grasscutter.game.world.Scene;
+import emu.grasscutter.server.event.types.ServerEvent;
+
+public final class SceneMetaLoadEvent extends ServerEvent {
+    public Scene scene;
+    public boolean hasOverride;
```

Comment:
consider making final or using `@Getter` and `@Setter`

Reply:
thanks; changed members to private and used @Getter/@Setter